src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/NameQuotedValuePairListEncoding.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.Designer;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+
+    [Export(typeof(INameValuePairListEncoding))]
+    [ExportMetadata("Encoding", "NameQuotedValuePairListEncoding")]
+    internal sealed class NameQuotedValuePairListEncoding : INameValuePairListEncoding
+    {
+        public IEnumerable<(string Name, string Value)> Parse(string value)
+        {
+            if (string.IsNullOrWhiteSpace(value))
+            {
+                yield break;
+            }
+
+            foreach (var entry in ReadEntries(value))
+            {
+                var (entryKey, entryValue) = SplitEntry(entry);
+
+                if (!string.IsNullOrEmpty(entryKey))
+                {
+                    yield return (DecodeCharacters(entryKey), StripQuotes(DecodeCharacters(entryValue)));
+                }
+            }
+
+            static string DecodeCharacters(string value)
+            {
+                return value.Replace("/=", "=").Replace("/,", ",").Replace("//", "/").Replace("/\"", "\"");
+            }
+        }
+
+        public string Format(IEnumerable<(string Name, string Value)> pairs)
+        {
+            return string.Join(",", pairs.Select(pair => $"{EncodeCharacters(pair.Name)}=\"{EncodeCharacters(pair.Value)}\""));
```

Comment:
If you're interested, this can be made to allocate fewer temporary objects by using a `StringBuilder` and appending to it as you walk through pairs. The `EncodeCharacters` method could also append to that.